pyimzml/ImzMLWriter.py

```diff
@@ -123,43 +133,69 @@ def __init__(self, output_filename, mz_dtype=np.float64, intensity_dtype=np.floa
         self.imzml_template = self.wheezy_engine.get_template('imzml')
 
         self.spectra = []
+        self.hashes = {} #set this to None to force processed mode.
```

Comment:
I'm not sure if it's a good idea. When it will be used with many datasets, hash collisions are bound to happen. Either m/z axis in case of a hit should be read from disk and checked for equality, or this whole business should be made explicit to the user e.g. by adding a subclass for continuous mode or a constructor parameter.